app/controllers/link_check_rules_controller.rb

```diff
@@ -36,6 +36,7 @@ def update
 
   private
     def link_check_rule_params
-      params.require(:link_check_rule).permit!
+      params.require(:link_check_rule).permit(:source_id, :xpath, :status_codes, :active, :throttle,
+                                              :collection_title, :_id, :created_at, :updated_at)
```

Comment:
Did you want to be permitted `created_at` and `updated_at`? Just checking this was intentional — it looks like it is but I don't know the context 😄

Reply:
Yeah it's kinda strange but we seem to send these params from the manager to the worker